src/js/view/view.js

```diff
@@ -607,9 +607,11 @@ function View(_api, _model) {
         if (state && _controls && model.get('autostartMuted')) {
             _controls.unmuteAutoplay(_api, model);
         }
+        const AMPController = _model._model.attributes.ampController;
```

Comment:
is there a reason we aren't using `_model.get('ampController')` here?

Reply:
Didn't know all model attribute refs were get-able. Good catch/info